meshroom/ui/reconstruction.py

```diff
@@ -496,6 +496,9 @@ def new(self, pipeline=None):
         elif p.lower() == "cameratracking":
             # default camera tracking pipeline
             self.setGraph(multiview.cameraTracking())
+        elif p.lower() == "photogrammetrynocuda":
```

Comment:
Would be better to name everything draft (instead of nocuda):
```suggestion
        elif p.lower() == "photogrammetrydraft":
```